### ISSUE TITLE
Uninstall from SQL Developer

### ISSUE THREAD

**Issue by rsngt** [jeffreykemp/jk64-plugin-reportmap#61]
Hi

I am just getting a blank region in my Apex app when I use the plug-in. I have tested in another Apex space on apex.oracle.com and it works fine. I've tested in multiple applications on my main Apex instance with different data sources, and also uninstalled and reinstalled the plugin, but I can't get it to work in my main instance.

I get a Javascript error:

_VM16687:4 Uncaught ReferenceError: reportmap is not defined
    at <anonymous>:4:1
    at r_R758xxxxxxxxxxx845 (f?p=146:2:153xxxxxxx25462::::::103)
    at <anonymous>:1:1_

I am wondering if this is caused by the installation I performed...

This was the first plug-in I installed in Apex, and I mistakenly installed it by running the install script from SQL Developer initially, before installing it via the Apex UI.

I've uninstalled and reinstalled via the Apex UI but still get the same error.

I wonder if it's caused by the rogue version that was installed via SQL Developer, and if I should uninstall that version, but aren't sure how to go about doing that.

Are you able to assist?

Thanks
Stephen

**Comment by jeffreykemp** [comment]
I doubt your problem was caused by running the script in SQL Developer. If you run the script directly from SQL Developer it will try to install the plugin in the workspace and application that it was exported from - the IDs are not going to match your workspace and application, so it won't install anything.

You can verify this by running "**select * from apex_appl_plugins**" which will list all the plugins that are installed.

Your javascript error will be caused by something different. I'm not sure what - but for whatever reason your application page has not loaded the reportmap plugin code. I would look for clues by examining the browser console log while running the page in debug mode. Another suggestion is to test this by creating a brand new application, install the plugin, and see if it works.

**Comment by rsngt** [comment]
Thanks for the reply. You are right, there is no record of the SQL Developer install, so that can't be the issue. I did create a new application in the same instance and receive the same error. As per my original post though, the app I built in the Oracle Apex cloud works fine, so it's obviously something in the instance, but not the data or the application itself. I have confirmed with the hosting provider that they have no restrictions around plugins.

Although it's pretty new to me, I will do some more debugging to see what is happening. I'll let you know what I find.
